pkg/core/snapshot.go

```diff
@@ -45,6 +45,9 @@ type Snapshot struct {
 	ReactMult        float64 //reaction multiplier for melt/vape
 	ReactBonus       float64 //reaction bonus %+ such as witch; should be 0 and only affected by hooks
 
+	//Special OHC flag
+	IsOHCDamage bool
```

Comment:
this is ok as a work around for now but with the reaction refactor there won't be a different apply damage function for reaction damage vs regular damage. i'll have to change this to be handled in the snapshot queued itself

Reply:
on that note, snapshot will need an IgnoreDefense flag for the purpose of.. well.. ignoring defense for reaction damage

Reply:
Yeah figured this would change with the refactor - not a big deal.